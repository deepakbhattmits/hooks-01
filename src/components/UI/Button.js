import React from 'react';
const Button = props => {
    const { type } = props
    return <button type={type}>{props.children}</button>
}
export default Button;