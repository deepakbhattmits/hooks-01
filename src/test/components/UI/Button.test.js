import React from "react";
import Button from "../../../components/UI/Button";
import { shallow } from "enzyme";
test("renders Button correctly", () => {
    const wrapper = shallow(<Button />);
    expect(wrapper.debug()).toMatchSnapshot();
});
