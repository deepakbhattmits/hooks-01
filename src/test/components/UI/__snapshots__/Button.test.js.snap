// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Button correctly 1`] = `"<button type={[undefined]} />"`;
